api/specs/web-server/_trash.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+from enum import Enum
+
+from fastapi import APIRouter, status
+from models_library.projects import ProjectID
+from simcore_service_webserver._meta import API_VTAG
+
+router = APIRouter(
+    prefix=f"/{API_VTAG}",
+    tags=["trash"],
+)
+
+
+@router.delete(
+    "/trash",
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+def empty_trash():
+    ...
+
+
+### Projects
```

Comment:
minor: what is this comment for?
the above is for trash, since there is `/trash` the ones below are for projects, since there is `/projects`, and also the tags.

Reply:
I had sections before, but not after.